src/lib/execution_plan_test.rs

```diff
@@ -315,6 +315,54 @@ cd -"#
     assert!(task.env.is_none());
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn create_workspace_task_with_included_members() {
```

Comment:
can we also have a test that tests the combination of both include and skip?

Reply:
Done.